app/ parsers/bulkrax/xml_etd_dc_parser.rb

```diff
@@ -18,6 +18,16 @@ def create_relationships
     end
 
     # needed to implement a passthrough as the xml parser doesn't support this yet in bulkrax.
-    def create_objects(type_array); end
+    def create_objects(type_array)
```

Comment:
What about file_sets?  In the `create_filesets` method we call `create_objects['file_set']` which wouldn't be handled.